### ISSUE TITLE
Internal Server error given when getting the members of list that has 5000 subscribers

### ISSUE THREAD

**Issue by xh3n1** [phpList/rest-api#117]
### Short problem description
I was testing  /lists/2/members for some lists, and I noticed that for a list that has more than 5000 subscribers, an internal server error is given.

**Comment by oliverklee** [comment]
@xh3n1 Is this a problem at exactly 5000 subscribers, i.e., 4999 work fine?

**Comment by xh3n1** [comment]
@oliverklee it's not exactly for 5000 subscribers, it gives the same error for 48720 or 54054.

**Comment by oliverklee** [comment]
Can you debug this? Maybe the PHP memory limit is reached.

**Comment by oliverklee** [comment]
And is the limit around 5000 or 50000 subscribers?

**Comment by xh3n1** [comment]
@oliverklee yes, I will debug this. The limit is around 50000 subscribers, sorry a typo.
